experimental/DoubleAndHyperComplexes/test/free_resolutions.jl

```diff
@@ -0,0 +1,16 @@
+@testset "free resolutions of ideals" begin
+  X = veronese();
+  I = defining_ideal(X);
+  Pn = base_ring(I)
+  FI = free_resolution(Oscar.SimpleFreeResolution, I);
+  F = graded_free_module(Pn, 1)
+  dualFIC = hom(FI, F);
+  # Duals invert the range from 0:10 to 0:-1:-10 etc. 
+  @test dualFIC[0] isa FreeMod
+  @test dualFIC[-1] isa FreeMod
+  @test dualFIC[-10] isa FreeMod
+  @test is_graded(dualFIC[0])
+  @test is_graded(dualFIC[-1])
+  @test is_graded(dualFIC[-10])
+  @test !Oscar.can_compute_index(dualFIC, 1)
```

Comment:
@wdecker : Your requested example is now up and running here. 

I was mistaken in that the `free_resolution` command also returns the augmentation map. I guess, I wanted to stay consistent with what `free_resolution` usually does and did not yet work out the augmentation map.

Reply:
Now the augmentation map is there.